user_docs/en/userGuide.t2t

```diff
@@ -519,6 +518,20 @@ When the menu comes up, You can use the arrow keys to navigate the menu, and the
 | Report clipboard text | NVDA+c | Reports the Text on the clipboard if there is any. |
 %kc:endInclude
 
+++ Speech modes ++[SpeechModes]
+
+There are four speech modes available.
+* Talk: NVDA will just speak normally. This is the default speech mode.
+* On-demand: NVDA will only speak when calling commands whose goal is to speak an information (e.g. report the title of the window); but it will not speak in reaction to actions such as moving the focus or the cursor.
+* Off: NVDA will not speak anything. 
+* Beeps: NVDA will replace normal speech with quiet beeps.

```

Comment:
I believe these lists also need a terminator. Is there a reason `*` was used over `-`?

```suggestion
* Beeps: NVDA will replace normal speech with quiet beeps.
*
```